### PR TITLE
Update README

### LINKED ISSUE #7: 2016年后社区的发展记录？

### PR DESCRIPTION
**Pull request by seisman**
Closes #7.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,13 +11,9 @@
...
-
-## 文件列表
-
-- `REAMDE.md` 本文件
-- `DNS.md` 记录DNS解析的相关设置
-- `domain-receipt.png` 域名收据的截图
-
+- 2017-03-10: [@seisman][seisman] 给域名续费一年，价格 ￥135.14
+- 2018-02-18: [@seisman][seisman] 给域名续费五年，价格 ￥675.70
+- 2019-01-27: [@seisman][seisman] 将模块手册和示例整理到中文手册中，废弃 modules 和 examples 子域名
```
Comment (core-man) [inline README.md:19]: Could we add when we created QQ groups 1 and 2?

Comment (seisman) [reply README.md:19]: Done in 6662ed7ca1a78265d53d5a5748755fa29f666b10

File: README.md [hunk 2]
```diff
@@ -9,13 +12,10 @@
...
-
-## 文件列表
-
-- `REAMDE.md` 本文件
-- `DNS.md` 记录DNS解析的相关设置
-- `domain-receipt.png` 域名收据的截图
-
+- 2017-03-10: [@seisman][seisman] 给域名续费一年，价格 ￥135.14
+- 2018-02-18: [@seisman][seisman] 给域名续费五年，价格 ￥675.70
+- 2018-08-13: 地学 GMT 中文社区 QQ 群 2 群（群号：791856541）建立
```
Comment (core-man) [inline README.md]:
```suggestion
- 2018-08-13: *地学 GMT 中文社区 2 群* QQ 群建立（群号：791856541）
```

File: README.md [hunk 3]
```diff
@@ -1,5 +1,8 @@
+# GMT 中文社区维护指南
+
 ## 社区发展时间线
 
+- 2012-09-14: 地学 GMT 中文社区 QQ 群 1 群（群号：218905582）建立
```
Comment (core-man) [inline README.md]:
```suggestion
- 2012-09-14: *地学 GMT 中文社区 1 群* QQ 群建立（群号：218905582）
```

File: README.md [hunk 4]
```diff
@@ -1,5 +1,8 @@
+# GMT 中文社区维护指南
+
 ## 社区发展时间线
 
+- 2012-09-14: *地学 GMT 中文社区 1 群* QQ 群建立（群号：218905582）
```
Comment (core-man) [inline README.md]: Bold is better?
```suggestion
- 2012-09-14: **地学 GMT 中文社区 1 群** QQ 群建立（群号：218905582）
```

File: README.md [hunk 5]
```diff
@@ -9,13 +12,10 @@
...
-
-## 文件列表
-
-- `REAMDE.md` 本文件
-- `DNS.md` 记录DNS解析的相关设置
-- `domain-receipt.png` 域名收据的截图
-
+- 2017-03-10: [@seisman][seisman] 给域名续费一年，价格 ￥135.14
+- 2018-02-18: [@seisman][seisman] 给域名续费五年，价格 ￥675.70
+- 2018-08-13: *地学 GMT 中文社区 2 群* QQ 群建立（群号：791856541）
```
Comment (core-man) [inline README.md]:
```suggestion
- 2018-08-13: **地学 GMT 中文社区 2 群** QQ 群建立（群号：791856541）
```

**Review by core-man — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -1,5 +1,8 @@
+# GMT 中文社区维护指南
+
 ## 社区发展时间线
 
+- 2012-09-14: *地学 GMT 中文社区 1 群* QQ 群建立（群号：218905582）
 - 2016-03-20: [@seisman][seisman] 建立组织 [@gmt-china](https://github.com/gmt-china)，建立 [gmt-china.github.io](http://gmt-china.github.io) 作为社区主页
```
Comment (core-man) [inline README.md]: gmt-china.github.io is dead.

Comment (core-man) [inline README.md]:
```suggestion
- 2016-03-20: [@seisman][seisman] 建立组织 [@gmt-china](https://github.com/gmt-china)，建立 `gmt-china.github.io` 作为社区主页
```

Comment (seisman) [reply README.md]: What do you mean?

Comment (core-man) [reply README.md]: Use `gmt-china.github.io`?

**Comment by core-man** [comment]
Good~ Please merge it.

**Pull request merged**